.github/workflows/build.yml

```diff
@@ -0,0 +1,68 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Last discussed, we should remove this and have staging image be promoted to production project.